src/util/bip324.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 The Bitcoin Core developers
```

Comment:
I'm not sure this belongs in util, it seems pretty specific to Bitcoin.

Reply:
I moved this into `net.{h|cpp}` as that's where it will be used. I didn't particularly want to increase the length of those files, so please let me know if the right thing to do is to have it in `src/bip324.{h|cpp}`